README.rst

```diff
@@ -181,12 +181,24 @@ are:
 
 - **fullrelease**: all of the above in order.
 
-There are two additional tools:
+There are some additional tools:
 
 - **longtest**: small tool that renders a setup.py's long description
   and opens it in a web browser. This assumes an installed docutils
   (as it needs ``rst2html.py``).
 
-- **lasttagdiff**: small tool that shows the diff of the currently committed
+- **lasttagdiff**: small tool that shows the *diff* of the currently committed
   trunk with the last released tag.  Handy for checking whether all the
   changes are adequately described in the changes file.
+
+- **lasttaglog**: small tool that shows the *log* of the currently committed
```

Comment:
Well, *something* is not right in the sentence :-) I think "with" is what's tripping me up. What is "the log of trunk with the tag"?